scripts/python-macro-helper

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python -tt
```

Comment:
Please don't use `-tt`, as that switch is deprecated and removed in Python 3.

Reply:
Shouldn't really matter all that much, because the script is invoked with an explicit "outside" interpreter without the shebang affecting anything, but meh... dropped.